spec/graphql/query/arguments_spec.rb

```diff
@@ -121,6 +121,33 @@
     end
   end
 
+  describe "#dig" do
+    it "returns the value at that key" do
+      assert_equal 1, arguments.dig("a")
+      assert_equal 1, arguments.dig(:a)
+      assert arguments.dig("inputObject").is_a?(GraphQL::Query::Arguments)
+    end
+
+    it "works with nested keys" do
+      assert_equal 3, arguments.dig("inputObject", "d")
+      assert_equal 3, arguments.dig(:inputObject, :d)
+      assert_equal 3, arguments.dig("inputObject", :d)
+      assert_equal 3, arguments.dig(:inputObject, "d")
+    end
+
+    it "returns nil for missing keys" do
+      assert_nil arguments.dig("z")
+      assert_nil arguments.dig(7)
+    end
+
+    it "handles underscored keys" do
+      # TODO shouldn't either of these work?
+      # assert_equal 3, arguments.dig('input_object', 'd')
+      # assert_equal 3, arguments.dig(:input_object, :d)
+    end
```

Comment:
Did you expect these to work, but they dont? Or, what do you mean by this comment?

Reply:
hah, no, there's no reason that should work, i was just copy-pasting. will remove.